collect_app/src/test/java/org/odk/collect/android/mainmenu/PermissionsDialogFragmentTest.kt

```diff
@@ -0,0 +1,78 @@
+package org.odk.collect.android.mainmenu
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.RootMatchers.isDialog
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.verify
+import org.odk.collect.android.R
+import org.odk.collect.android.fakes.FakePermissionsProvider
+import org.odk.collect.androidshared.ui.FragmentFactoryBuilder
+import org.odk.collect.fragmentstest.FragmentScenarioLauncherRule
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(sdk = [33])
```

Comment:
This is redundant since by default robolectric runs our code against the targetSdkVersion specified in the manifest. So it will never be lower than 33.